config.go

```diff
@@ -271,6 +271,8 @@ type Config struct {
 
 	ResetWalletTransactions bool `long:"reset-wallet-transactions" description:"Removes all transaction history from the on-chain wallet on startup, forcing a full chain rescan starting at the wallet's birthday. Implements the same functionality as btcwallet's dropwtxmgr command. Should be set to false after successful execution to avoid rescanning on every restart of lnd."`
 
+	CoinSelectionStrategy string `long:"coin-selection-strategy" description:"The strategy to use for selecting coins." choice:"largest" choice:"random"`
```

Comment:
Needs an entry in the `sample-lnd.conf` file.

Reply:
Should note that this strategy will only be used for transactions originating from the wallet and not channels, sweeps, etc. Is there any reason why you didn't include the strategy for those?

Reply:
The reason that I didn't include it for those is that it is a larger and more risky refactoring. For channels and sweeps, lnd has its own coin selection logic and doesn't use the code in btcwallet. I think it is worth to move towards consolidating that, but didn't want to do it in this PR.

I think that random coin selection is mainly important for users that have high volumes of wallet transactions. Lightning channels should be long-lived, so coin selection is arguably less critical.

Updated description to make it clear that the strategy only applies to wallet transactions.

Added flag to `sample-lnd.conf`.